sdk_extensions/logging/support/src/main/java/io/opentelemetry/sdk/logging/LogSinkSdkProvider.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.logging;
+
+import io.opentelemetry.internal.Utils;
+import io.opentelemetry.sdk.logging.data.LogRecord;
+import java.util.ArrayList;
+import java.util.List;
+
+public class LogSinkSdkProvider {
+  private final LogSink logSink = new SdkLogSink();
+  private final List<LogProcessor> processors = new ArrayList<>();
+
+  private LogSinkSdkProvider() {}
+
+  public LogSink get(String instrumentationName, String instrumentationVersion) {
+    // Currently there is no differentiation by instrumentation library
+    return logSink;
+  }
+
+  public void addLogProcessor(LogProcessor processor) {
+    processors.add(Utils.checkNotNull(processor, "Processor can not be null"));
+  }
+
+  /** Flushes all attached processors. */
+  public void forceFlush() {
```

Comment:
Ditto

Reply:
Ouf. I think this one was actually hiding a bug where I wouldn't have finished flushing before shutting down the exporter.